python/python/glide/async_commands/stream.py

```diff
@@ -363,6 +363,69 @@ def __init__(
         self.consumer_name = consumer_name
 
 
+class StreamClaimOptions:
+    IDLE_REDIS_API = "IDLE"
+    TIME_REDIS_API = "TIME"
+    RETRY_COUNT_REDIS_API = "RETRYCOUNT"
+    FORCE_REDIS_API = "FORCE"
+    JUST_ID_REDIS_API = "JUSTID"
```

Comment:
nit
```suggestion
    JUSTID_REDIS_API = "JUSTID"
```

Reply:
Please double confirm as it is a straight copy from [`java/client/src/main/java/glide/api/models/commands/stream/StreamClaimOptions.java#L31`](https://github.com/Bit-Quill/glide-for-redis/blob/867cd9b2277b1e3a07f22c5cc26547ee2dc3f83e/java/client/src/main/java/glide/api/models/commands/stream/StreamClaimOptions.java#L31)

Reply:
Sorry... I missed it before.  Again, not worthwhile probably.